src/omv/imlib/draw.c

```diff
@@ -89,42 +89,46 @@ static void point_fill(image_t *img, int cx, int cy, int r0, int r1, int c) {
 }
 
 static void imlib_set_pixel_aa(image_t *img, int x, int y, int err, int c) {
-    // float alpha = 1 - err / 256.0;
```

Comment:
Can we instead check and return ?

```C
if (!((0 <= x) && (x < img->w) && (0 <= y) && (y < img->h))) {
    // Out of bounds.
    return;
}

Reply:
Sure, yeah. That's faster.

Reply:
Updated.

Reply:
Note, I removed imlib_set_pixel() because that would bounds check again.